### PR TITLE
BugFix: Undetermined flowcells again (total number was wrong)

### PR DESCRIPTION
**Pull request by kate-v-stepanova**
sorry, one more mistake

### CODE REVIEW

**Comment by ewels** [comment]
Conflicts :confused: Did you do some kind of rebasing again? It will create conflicts if so as we already merged your previous changes.

**Comment by Galithil** [comment]
Also, I believe this eats the first element that's being iterated upon.

File: run_dir/static/js/flowcell_samples.js [hunk 1]
```diff
@@ -192,14 +192,13 @@ $.getJSON("/api/v1/flowcell_info2/"+flowcell, function(data) {
...
-                    if(total == -1) { percentage = '100.00' };
                     count = nice_numbers(count);
                     if(total == -1) {
-                        ludtable += "<tr><th>"+unmatched+"</th><th>"+nice_numbers(total_undetermined_claster_number)+'</th><th>('+percentage+"%)</span></th></tr>";
+                        ludtable += "<tr><th>Total</th><th>"+nice_numbers(total_undetermined_claster_number)+"</th><th>(100%)</span></th></tr>";
                         total = parseInt(data['undetermined'][lid][ordered_keys[ud]]);
-                    } else {
-                        ludtable += "<tr"+hl+"><td><samp>"+unmatched+"</samp></td><td>"+count+'</td><td>('+percentage+"%)</span></td></tr>";
                     }
+                    else {ludtable += "<tr"+hl+"><td><samp>"+unmatched+"</samp></td><td>"+count+'</td><td>('+percentage+"%)</span></td></tr>";}
```
Comment (ewels) [inline run_dir/static/js/flowcell_samples.js]: I think @Galithil is suggesting that you don't need the `else` here..

**Comment by kate-v-stepanova** [comment]
I think I just forgot to pull origin before pushing, that's why I had this merge conflict. Now should be fine

**Comment by ewels** [comment]
I'm not sure that you want this change though? It looks like it would still eat the first element.. @Galithil, do you agree?

**Comment by kate-v-stepanova** [comment]
there was an "else" missing, that's why the first element was overwritten. so i added 'else'

**Comment by kate-v-stepanova** [comment]
yes, right. instead of the first element a header was displayed. but we have to show both. now should be correct

**Comment by ewels** [comment]
That's better! :)

**Pull request merged**